nixos/doc/manual/configuration/gpu-accel.xml

```diff
@@ -184,6 +184,8 @@ GPU1:
 
 	<programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = [
   <package>amdvlk</package>
+  # For the 32-bit driver
+  <package>driversi686Linux.amdvlk</package>
```

Comment:
This should go in `hardware.opengl.extraPackages32`, right? Or doesn't the ICD loader pick it up there?

Reply:
Ah, I didn’t know there was a 32-bit option too. Seems to work fine, 32-bit vkcube chose radv for me though 32-bit vulkaninfo showed both, radv and amdvlk.